pkg/controller/servicebindingrequest/servicebinder_test.go

```diff
@@ -573,3 +572,27 @@ func TestServiceBinder_Bind(t *testing.T) {
 		},
 	}))
 }
+
+func TestEnsureDefaults(t *testing.T) {
+	t.Run("label selector with non nil", func(t *testing.T) {
+		applicationSelector := &v1alpha1.ApplicationSelector{}
+		applicationSelector.LabelSelector = &metav1.LabelSelector{}
+		ensureDefaults(applicationSelector)
+		require.NotNil(t, applicationSelector.LabelSelector)
```

Comment:
`applicationSelector.LabelSelector` is already not `nil`, what is expected in this test?

Reply:
Fixed. 290162d1885dc6688870528113cb13d07cf80a60